<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QToolBox" name="toolBox">
    <property name="geometry">
     <rect>
      <x>1550</x>
      <y>10</y>
      <width>351</width>
      <height>511</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="page">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>351</width>
       <height>406</height>
      </rect>
     </property>
     <attribute name="label">
      <string>软件管理</string>
     </attribute>
     <widget class="QPushButton" name="pButton_AddMap">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>10</y>
        <width>70</width>
        <height>70</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">image: url(:/icon/icon/addmap.svg);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pButton_Setting">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>10</y>
        <width>70</width>
        <height>70</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">image: url(:/icon/icon/setting.svg);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pButton_exit">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>10</y>
        <width>70</width>
        <height>70</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">image: url(:/icon/icon/close.svg);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pButton_MapManager">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>110</y>
        <width>70</width>
        <height>70</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">image: url(:/icon/icon/MapManager.svg);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>351</width>
       <height>406</height>
      </rect>
     </property>
     <attribute name="label">
      <string>地图工具</string>
     </attribute>
     <widget class="QPushButton" name="pButton_Mark">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>10</y>
        <width>70</width>
        <height>70</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">image: url(:/icon/icon/mark.svg);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pButton_Zoomup">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>10</y>
        <width>70</width>
        <height>70</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">image: url(:/icon/icon/enlarge.svg);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pButton_Zoomdown">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>10</y>
        <width>70</width>
        <height>70</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">image: url(:/icon/icon/lessen.svg);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pButton_Measure">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>120</y>
        <width>70</width>
        <height>70</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">image: url(:/icon/icon/measure.svg);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>351</width>
       <height>406</height>
      </rect>
     </property>
     <attribute name="label">
      <string>路线规划</string>
     </attribute>
     <widget class="QPushButton" name="pButton_TPlan">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>10</y>
        <width>70</width>
        <height>70</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">image: url(:/icon/icon/taskplan.svg);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>1550</x>
      <y>550</y>
      <width>351</width>
      <height>461</height>
     </rect>
    </property>
    <property name="title">
     <string>态势管理</string>
    </property>
    <widget class="QPushButton" name="pButton_MShip">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>45</y>
       <width>70</width>
       <height>70</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/icon/icon/mship.svg);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pButton_SShip">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>240</y>
       <width>70</width>
       <height>70</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/icon/icon/sship.svg);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>220</y>
       <width>210</width>
       <height>119</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>经度</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEdit_4">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>118</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>纬度</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEdit_5">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>31</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>航向</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="lineEdit_6">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>180</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>航速</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="lineEdit_8">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>3.0</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>30</y>
       <width>210</width>
       <height>119</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>经度</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEdit">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>118</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>纬度</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEdit_2">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>31</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>航向</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="lineEdit_3">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>180</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>航速</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="lineEdit_7">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>3.0</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="InteractiveMap" name="mapgrapview">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1551</width>
      <height>791</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1920</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>InteractiveMap</class>
   <extends>QGraphicsView</extends>
   <header>interactivemap.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>pButton_exit</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1834</x>
     <y>124</y>
    </hint>
    <hint type="destinationlabel">
     <x>959</x>
     <y>539</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
